src/miniclient/client.go

```diff
@@ -81,7 +85,8 @@ func (mm *Conn) Pipe(pipe string) (io.Reader, io.WriteCloser) {
 		PlumbPipe: pipe,
 	})
 	if err != nil {
-		log.Fatal("local pipe gob encode: %v", err)
+		mm.err = fmt.Errorf("local pipe gob encode: %v", err)
```

Comment:
I didn't see a Lock() prior to writing this; is Pipe() only ever called after something else has locked mm?